### PR TITLE
Feat: Lido Moonbeam

### PR DESCRIPTION
**Pull request by Oladiy**
**NOTE**

1. The protocol is usually listed within 24 hours of merging the PR
2. Please fill the form below  **only if the PR is for listing a new protocol** else it can be ignored/replaced with reason/details about the PR
3. **For updating listing info** It is a different repo, you can find your listing in this file: https://github.com/DefiLlama/defillama-server/blob/master/defi/src/protocols/data2.ts, you can  edit it there and put up a PR
4. Do not edit/push `package-lock.json` file as part of your changes, we use lockfileVersion 2, and most use v1 and using that messes up our CI

---

### CODE REVIEW

**Comment by llamatester** [comment]
The adapter at projects/lido exports TVL: 
        
 
 
    terra                     8.57 k
    moonriver                 1.89 M
    moonbeam                  3.07 M
    ethereum                  5.19 B
    solana                    89.07 M
    
    total                    5.29 B

File: projects/lido/index.js [hunk 1]
```diff
@@ -58,6 +58,21 @@ async function ksm(timestamp, ethBlock, chainBlocks) {
   }
 }
 
+async function dot(timestamp, ethBlock, chainBlocks) {
+  const chain = "moonbeam"
+  const block = await getBlock(timestamp, chain, chainBlocks, true)
+  const pooledCoin = await sdk.api.abi.call({
+    block,
+    chain,
+    target: "0xFA36Fe1dA08C89eC72Ea1F0143a35bFd5DAea108",
```
Comment (karapetyantv) [inline projects/lido/index.js]: Convert address to lower case for consistency with other addresses

**Review by karapetyantv — CHANGES_REQUESTED** [review]

**Comment by llamatester** [comment]
Error while running adapter at projects/lido: 
        
 
  
    
    
    Error: 503 Service Unavailable: no available upstreams for 'validator' category
    Truncated error stack:
        at ClientBrowser.callServer (/home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/node_modules/@solana/web3.js/lib/index.cjs.js:5016:18)

File: projects/lido/index.js (hunk 1 above)
Comment (Oladiy) [reply projects/lido/index.js]: > Convert address to lower case for consistency with other addresses

Done

**Comment by llamatester** [comment]
Error while running adapter at projects/lido: 
        
 
  
    
    
    Error: 503 Service Unavailable: no available upstreams for 'validator' category
    Truncated error stack:
        at ClientBrowser.callServer (/home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/node_modules/@solana/web3.js/lib/index.cjs.js:5016:18)

**Pull request merged**